workers/setup.cfg

```
@@ -12,13 +11,14 @@ inline-quotes = double
 max-line-length = 88
 
 per-file-ignores =
-    # TODO: Fix issues causing those warnings
-    ohsome_quality_analyst/indicators/mapping_saturation/sigmoid_curve.py:N806,N802,N803,N801
     # N803: argument name should be lowercase
-    #   Ignore for Fast-API parameters which are defined as mixedCase
     # N815: mixedCase variable in class scope (constants, method names)
-    #   Ignore for Fast-API parameters which are defined as mixedCase
+    # N806: variable in function should be lowercase
+    # N815: mixedCase variable in class scope (constants, method names)
+    # Ignore for Fast-API parameters which are defined as mixedCase
     ohsome_quality_analyst/api/api.py:N803,N815
```

Is N815 still necessary?
```suggestion
    # Ignore for Fast-API parameters which are defined as mixedCase
    ohsome_quality_analyst/api/api.py:N803
```

---

Good catch. Its not. I removed it.